cmd/fips/fips_test.go

```diff
@@ -0,0 +1,27 @@
+package main_test
+
+import (
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestFips(t *testing.T) {
+	cmd := exec.Command("go", "run", ".", "./testdata")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Logf("fips output:\n%s", output)
+		t.Fatal(err)
+	}
+	got := strings.Split(strings.ReplaceAll(string(output), "\r\n", "\n"), "\n")
+	got = got[:len(got)-1] // remove last empty line jump
+	want := []string{
```

Comment:
Is `got / want` a common idiom in Go testing code? Typically used to `actual / expected` in other langs.

Reply:
Yes it is. The tooling that automatically create tabular tests for a given function uses this idiom. Also, searching for `got := ` in the Go source code yields 1265 results in 366 files, and `actual := ` only 32 results in 15 files... probably from someone coming from another lang ;)

Reply:
What tooling are you using to generate the tests?

Reply:
I believe VS Code and maybe GoLand use https://github.com/cweill/gotests to generate them. You can see `got`/`want` in some official docs here, too: https://golang.org/doc/code#Testing

Reply:
VSCode+Go extension provides this capability out of the box. I think https://github.com/cweill/gotests is being used under the hood.

Reply:
@dagood didn't see your response until I already submitted mine 😄